BWEnv/src/controller.cc

```diff
@@ -261,9 +261,14 @@ void Controller::handleCommand(int command, const std::vector<int>& args,
       return;
     }
   }
+  else if (command == Commands::SET_MAP) {  
+	  // Due to the flatbuffers change, set_map is noargs, but we don't want to break backward compat.
```

Comment:
We have significantly changed things underneath, so I'd be actually OK with breaking some of the interface (or at least print some sort of deprecation warning in the clients when `setMap` is used).